Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallContentCommand.cs

```diff
@@ -187,30 +189,44 @@ private void InstallFile(SourceFile sourceFile, CancellationToken cancellationTo
                 DebugLogger.LogFormat("Destination file {0} already exists, removing it.", destinationFilePath);
                 FileOperations.Delete(destinationFilePath, cancellationToken);
             }
-
-            FileOperations.Move(sourceFile.FullPath, destinationFilePath, cancellationToken);
+#if UNITY_STANDALONE_WIN
+            if (destinationFilePath.Length > 259)
```

Comment:
Did you check (practically) if 260 characters is the limit? AFAIK it could be somewhere between 255 and 260. Please try this in practice.

Reply:
I tried this in practice and this is result